### PR TITLE
Yuval/check dir size

### PR DESCRIPTION
**Pull request by yuval-lb**
following: https://lightbitslabs.atlassian.net/browse/LBM1-17330
Machine is inaugurated without available storage

### CODE REVIEW

File: inaugurator/dirsize.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import os
+ 
+def get_dir_size_used(dest_root_path):                        
+    st = os.statvfs(dest_root_path)
```
Comment (yant-lb) [inline inaugurator/dirsize.py:4]: As a sanity check, maybe it is better to try & except it? as `os.statvfs` throws `FileNotFoundError` when dir doesn't exist.

File: inaugurator/dirsize.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    used_precent = float((st.f_blocks - st.f_bavail)) / st.f_blocks
```
Comment (yant-lb) [inline inaugurator/dirsize.py:5]: Also try except the division as `f_blocks` might in some cases be zero :-D

**Comment by yuval-lb** [comment]
@yant-lb  updated

**Review by yant-lb — APPROVED** [review]
Great, thanks.

**Comment by yuval-lb** [comment]
this was tested before on client 05-75
before new inaugurator
/dev/inaugurator-v2/root
                          2.0E      9.5E         0 100% /destRoot (100% use)

after:
 0% /destRoot

**Pull request merged**